README.md

```diff
@@ -8,6 +8,12 @@ Workspace for a Rust wrapper for SuiteSparse:GraphBLAS. The workspace contains t
 ## Dependencies
 graphblas_sparse_linear_algebra uses the [SuiteSparse:GraphBLAS](https://github.com/DrTimothyAldenDavis/GraphBLAS) GraphBLAS implementation developed by Timothy A. Davis.
 
+To build and test on Ubuntu 24.04, [install Rust](https://www.rust-lang.org/tools/install) and ensure the dependencies are installed:
```

Comment:
graphblas_sparse_linear_algebra is a library crate and would normally be added as a dependency of another Rust application. Adding a reference to the installation instructions for Rust itself should be redundant for the target user group. It's uncommon to add build instructions for the host language in a library crate.